tests/functional/firefox/test_developer.py

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from ...pages.firefox.developer import DeveloperPage
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_download_buttons_are_displayed(base_url, selenium):
+    page = DeveloperPage(base_url, selenium).open()
+    assert page.is_primary_download_button_displayed
+    assert page.is_secondary_download_button_displayed
+
+
+# Skipping this test until bug 1219251 is fixed.
+# https://bugzilla.mozilla.org/show_bug.cgi?id=1219251
+# @pytest.mark.sanity
+# @pytest.mark.nondestructive
+# def test_play_video(base_url, selenium):
+#     page = DeveloperPage(base_url, selenium).open()
+#     video = page.developer_videos[0]
+#     video.play()
+#     assert video.modal_is_displayed
+#     video.close()
+
+
+@pytest.mark.nondestructive
+def test_newsletter_default_values(base_url, selenium):
```

Comment:
Is this another unique newsletter? If not, how many pages are we going to test the newsletter from? I wonder if we should have some newsletter tests that are parametrised by page.